packages/entities/entities-shared/src/index.ts

```diff
@@ -12,13 +12,13 @@ import EntityFormSection from './components/entity-form-section/EntityFormSectio
 import composables from './composables'
 
 // Extract specific composables to export
-const { useAxios, useDeleteUrlBuilder, useErrors, useFetchUrlBuilder, useFetcher, useDebouncedFilter, useStringHelpers, useHelpers, useGatewayFeatureSupported } = composables
+const { useAxios, useDeleteUrlBuilder, useErrors, useFetchUrlBuilder, useFetcher, useDebouncedFilter, useStringHelpers, useHelpers, useGatewayFeatureSupported, useTruncationDetector } = composables
 
 // Components
 export { EntityBaseConfigCard, ConfigCardItem, ConfigCardDisplay, InternalLinkItem, EntityBaseForm, EntityBaseTable, EntityDeleteModal, EntityFilter, EntityToggleModal, PermissionsWrapper, EntityFormSection }
 
 // Composables
-export { useAxios, useDeleteUrlBuilder, useErrors, useFetchUrlBuilder, useFetcher, useDebouncedFilter, useStringHelpers, useHelpers, useGatewayFeatureSupported }
+export { useAxios, useDeleteUrlBuilder, useErrors, useFetchUrlBuilder, useFetcher, useDebouncedFilter, useStringHelpers, useHelpers, useGatewayFeatureSupported, useTruncationDetector }
```

Comment:
question: since the truncation composable is only used in this package, why does it need to be exported?

Reply:
I'm using it in `khcp-ui`

Reply:
Should that be a helper composable exported from https://github.com/Kong/public-ui-components/tree/main/packages/core/core instead?

Reply:
Meh nevermind; keep as-is 🤷🏼